packages/api-specification/src/hiddenWindow.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
Yeah it was in api-specification to avoid the duplication, but I could move it if you think that's better? I think _hiddenMainWindow_ is probably my preference from those.